CHANGELOG.md

```diff
@@ -24,6 +24,8 @@ This project adheres to [Semantic Versioning](https://semver.org/).
   ([#1833](https://github.com/nix-rust/nix/pull/1833))
 - Added `faccessat(2)` on illumos
   ([#1841](https://github.com/nix-rust/nix/pull/1841))
+- Added `eaccess()` on freeBSD, dragonfly and Linux (glibc and musl).
```

Comment:
Nit: can you style them as `FreeBSD` and `DragonFly` throughout? I don't think we're 100% consistent right now, but we should try to be moving forward.

Reply:
Absolutely, done!